wallet_ui/utils/cycles.ts

```diff
@@ -1,13 +1,32 @@
-export function format_cycles(cycles : bigint) {
-  const [cycles_string, suffix] = format_cycles_and_suffix(cycles);
-  return cycles_string + " " + suffix + "C";
+export function format_cycles_trillion(cycles: bigint, fixed: number) {
+  const trillion = 1000000000000;
+  const cyclesInTrillion = parseFloat(cycles.toString()) / trillion;
+  const cycles_string =
+    cyclesInTrillion % 1 === 0
+      ? cyclesInTrillion
+      : cyclesInTrillion.toFixed(fixed);
+  return cycles_string.toLocaleString();
 }
 
-export function format_cycles_and_suffix(cycles : bigint) {
+export function format_cycles_trillion_fullrange(cycles: bigint) {
   const cycles_float = parseFloat(cycles.toString());
-  const SUFFIX_LIST = " KMGTPE";
-  const suffix = SUFFIX_LIST[Math.min(6, Math.floor(Math.log10(cycles_float) / 3))];
-  const ll = SUFFIX_LIST.indexOf(suffix);
-  const humanCycles = Math.floor(cycles_float / 10 ** (ll * 3));
-  return [humanCycles.toString(), suffix];
-}  
+  const units = {
```

Comment:
Are you still using these units? I recall you saying you were going to go with TC across the board

Reply:
No I'm not using those units anymore. They are replaced with different standard of decimals.